.github/workflows/build.yaml

```diff
@@ -661,7 +662,8 @@ jobs:
             "collector",
             "collector-slim",
             "main",
-            "roxctl",
+            # TODO(stehessel): re-enable roxctl scanning after #10928 has been released.
```

Comment:
Let's make this a ticket marker just for posterity (also in case someone else picks this up besides you).